### ISSUE TITLE
DevOpsDays 2016 - Volunteer Kickoff

### ISSUE THREAD

**Issue by jfryman** [DevOpsDaysBNA/discussions#4]
Hello all!

TL;DR: Thanks for volunteering! If you are still interested in helping, please send me your GitHub user ID, and I will add you to the GitHub org to get everything started.

I am writing today as at one point or another, you have expressed interest in helping out with making a DevOpsDays happen here in Nashville.

Well, if you haven’t heard, we’re *one* step closer to getting the conference off the ground. We’ve spun up conversations with the DevOpsDays Leadership Committee, organized a core leadership group here in town, and are starting to get our ducks in order. To that end, I would like to reach out to all of you, thank you for offering your help and support, and let you know where we are.

http://www.devopsdays.org/events/2016-nashville/

CHECK IT OUT!!! ^^^

First off, DevOpsDays BNA Core team. For the purposes of final decision making, we have these wonderful volunteers who are helping lead the charge to bring DevOpsDays here. They are (name and company)

* Chris Wage - OpenSky
* Eliza Marcum - EBS
* James Fryman - auth0
* Jurnell Cockhren - SophicWare
* Nick Wallace - EMC

These folks will be helping organize various parts of the DevOpsDays. We’re going to try something new, and have a completely open DevOpsDays organization. All discussions will happen in public on GitHub, and any planned meeting will be recorded and public. DevOps is about openness and inclusiveness. So, we have leaders to help with final decisions, but it should be possible as a volunteer to see and be a part of planning DevOpsDays BNA 2016.

What can you do today?

A few things. We are several months away from actually hosting this thing (currently soft-targeting Nov 11). We have a lot to do between now and then, and we might be able to use your help.

1) Tell folks. Tell your co-workers it’s coming, get them excited. Tell user groups you attend, get them excited. Tell your friends, get them excited. This will be our first *of many* DevOpsDays here in Nashville. But, the first conference always has a bit of special charm about it. Let folks know it’s real, encourage them to tell their friends, see if their companies want to help contribute to *the very first DevOpsDays* here in Nashville
2) Tell folks. Yeah, again. :) See how folks in the community are making it easier to ship software between Dev and Ops groups. Find cool stories, and encourage folks to write about it. There are Nashville stories to be told, let’s go get them.
3) Tell your employers. Maybe they want to help make this a reality! Sponsorship with money, or space, or we can even get creative.
4) Find other volunteers. As you talk about bringing DevOpsDays

and the most important…

Find Local Speakers. We want to have a wonderful showing for our first conf here in town, so encourage folks to be thinking about papers. A general CFP will be put out as soon as we confirm building space and dates. But, it’s coming soon.

For the most part, between now and the conference your help will be in getting the word out. We’ll have plenty of opportunities to help out between now and conf time, and more than enough opportunities to help out the weeks prior the conference. For now, I encourage you all to keep a watch on the DevOpsDay GitHub org. You can see that at https://github.com/DevOpsDaysBNA/discussions. We will keep everything up there, so feel free to provide input or volunteer for bits of work as they come up. This is going to be a truly collaborative process.

Thank you so much for your help, and I look forward to working with you all!

-James
